at_secondary/at_persistence_secondary_server/lib/src/keystore/hive_keystore.dart

```diff
@@ -381,4 +407,8 @@ class HiveKeystore implements SecondaryKeyStore<String, AtData?, AtMetaData?> {
       await persistenceManager.openVault(persistenceManager.atsign!);
     }
   }
+
+  bool isExpired(key) {
+    return _metaDataCache[key]!.expiresAt!.isBefore(DateTime.now().toUtc());
```

Comment:
- Check where expiresAt is being set
- If it is set on server, and if the key happens to be created and exired when the device is offline then expiredAt would have not been set. We have to come up with a solution for this.